rocAL/rocAL/source/augmentations/geometry_augmentations/node_resize_mirror_normalize.cpp

```diff
@@ -85,18 +85,15 @@ void ResizeMirrorNormalizeNode::create_node()
 
 void ResizeMirrorNormalizeNode::update_node()
 {
-    std::vector<uint32_t> src_roi_width, src_roi_height;
-    src_roi_width = _inputs[0]->info().get_roi_width_vec();
-    src_roi_height = _inputs[0]->info().get_roi_height_vec();
-
+    RocalROI* src_roi = _inputs[0]->info().get_roi();   // Check if it needs to be similar to resize
     for(uint i = 0; i < _batch_size; i++)
     {
         // Min size and max size used for MLPerf MaskRCNN resize augmentation
         // TODO: Get the min_size and max_size as user arguments from python
         int min_size = 800;
         int max_size = 1333;
-        int src_width = src_roi_width[i];
-        int src_height = src_roi_height[i];
+        int src_width = src_roi[i].x2;
```

Comment:
I think for resize we are doing updation in 1 method and for RMN, we are doing updation in another method
Please check if both can be done in same way

Reply:
RMN logic works specifically for Mask training will have to fix this in future